thirdeye-server/src/test/java/ai/startree/thirdeye/resources/AlertResourceTest.java

```diff
@@ -300,7 +306,10 @@ public void testEvaluate_withExistingAlertAndReadAccessToAlertAndPartialAccessTo
         mock(AlertInsightsProvider.class),
         new AuthorizationManager(alertTemplateRenderer,
```

Comment:
seems like we need different instances of `authorizationManager`?

probably better to have `createAuthorizationManager(accessControl)` then? 
Seems like 1 single arg is changing. rest are all the same. 

Also eventually, you may need mocks here.